packages/shipjs/src/step/prepare/__tests__/updateChangelog.spec.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+import { prepareParams } from '../updateChangelog';
+import { parseArgs } from '../../../util';
+import tempWrite from 'temp-write';
+
+describe('prepareParams', () => {
+  it('loads configuration from --config option', () => {
+    parseArgs.mockImplementation(jest.requireActual('../../../util').parseArgs);
```

Comment:
I had to restore the original implementation of `parseArgs` because I made a wrong choice in the beginning of Ship.js to globally mock many things including `util`. It's so hidden and hard to find out things like this :(

Reply:
Yes something was wrong with the `parseArgs` but I just discovered this morning that you finished the PR.

Just to understand: what does this test mock exactly? It seems we are mocking `parseArgs` but at the end, the config seems correctly generated, how does that work?

Reply:
Actually `parseArgs` was not supposed to be mocked, but I mocked everything under `util` globally, which made `parseArgs` returned nothing. So it made the test case failed. This line `parseArgs.mockImplementation(jest.requireActual('../../../util').parseArgs);` is restoring its original implementation.

Reply:
Got it, that's why it was returning me `undefined` and I couldn't see the `console.log()` I've added in its implementation! Thanks for letting me know. (my 2 cents on that, you should probably do it the other way around when testing: avoid to use global mocking, and use it explicitely when you need it in your tests).